src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -3108,6 +3115,28 @@ int Pane::GetLeafPaneCount() const noexcept
     return _IsLeaf() ? 1 : (_firstChild->GetLeafPaneCount() + _secondChild->GetLeafPaneCount());
 }
 
+// Method Description:
+// - Should be called when this pane is created via a default terminal handoff
+// - Finalizes our configuration given the information that we have been
+//   created via default handoff
+void Pane::FinalizeConfigurationGivenDefault()

```

Comment:
Right now this function only overrides closeOnExit, but I implemented it like this for any future scenario where we might want to configure some other things as well if we know this was a defterm invocation